symbulate/distributions.py

```diff
@@ -135,14 +135,14 @@ def __init__(self, n, N0, N1):
         if n > 0 and (isinstance(n, int)):
             self.n = n
         else:
-            raise Exception("Number of draws must be an integer and cannot be negative")
+            raise Exception("Number of draws must be a non-zero integer and cannot be negative")
```

Comment:
Number of *draws* is confusing because Symbulate has a .draw() function. Just say n.